src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Default {
+    private double size;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double size, double firstLeg, double secondLeg) {
+        super(color);
+        this.size = size;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+
+        return size * size;
+    }
+
+    @Override
+    public double getProperty() {
```

Comment:
The method name `getProperty()` is not descriptive. Consider renaming it to something more meaningful like `getPerimeter()` if it calculates the perimeter, or another appropriate name.